sdk/typescript/runtime/main.go

```diff
@@ -90,9 +106,13 @@ func (t *TypescriptSdk) ModuleRuntime(ctx context.Context, modSource *ModuleSour
 			WithExec([]string{"bun", "install", "--no-verify", "--no-progress"}).
 			WithEntrypoint([]string{"bun", entrypointPath}), nil
 	case Node:
+		// Install dependencies
+		ctr, err = installNodeDependencies(ctx, ctr, packageManager)
+		if err != nil {
+			return nil, fmt.Errorf("failed to install dependencies: %w", err)
+		}
```

Comment:
Looks like `installNodeDependencies` supports `BunManager` so the dependencies install can be moved out of the switch.

Reply:
True